### PR TITLE
chore: fix console warnings

### PR DESCRIPTION
**Pull request by aswathkk**
## Description

Fixes the following warnings that we observe on the console.

Screenshot 2023-02-13 at 1 18 54 PM
Screenshot 2023-02-13 at 1 19 30 PM

## Type of change

> Please delete options that are not relevant.

- Chore (housekeeping or task changes that don't impact user perception)

- Manual
Tested by ensuring the Tabs and Switch on the property pane are working as expected.

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx [hunk 1]
```diff
@@ -120,6 +120,8 @@ export function PropertyPaneTab(props: PropertyPaneTabProps) {
             </Tab>
           )}
         </TabList>
+        {props.contentComponent && <TabPanel />}
+        {props.styleComponent && <TabPanel />}
```
Comment (Rishabh-Rathod) [inline app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx:124]:
```suggestion
        {(props.contentComponent  || props.styleComponent) && <TabPanel />}
```

Comment (aswathkk) [reply app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx:124]: We need to have 2 TabPanel if both content & style component exists

**Review by Rishabh-Rathod — APPROVED** [review]
LGTM

File: app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx (hunk 1 above)
Comment (Rishabh-Rathod) [reply app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx:124]: Sorry, my bad.

**Pull request merged**